src/redux/stock/actionCreator.js

```diff
@@ -0,0 +1,40 @@
+import { API_URL, API_KEY } from 'react-native-dotenv';
+
+import { FETCH_STOCK_DETAILS_START, FETCH_STOCK_DETAILS_FAIL, FETCH_STOCK_DETAILS_SUCCESS } from './actionType';
+
+const fetchStockDetailsStart = () => ({
+  type: FETCH_STOCK_DETAILS_START,
+});
+
+const fetchStockDetailsFail = (err) => ({
```

Comment:
if you call the argument `payload` then you can use the shorthand property names

Reply:
Updated